Sources/Utils/Rng/MobileCoinChaCha20Rng.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import Foundation
+import LibMobileCoin
+
+final class MobileCoinChaCha20Rng: MobileCoinSeedableRng {
+    private var ptr: OpaquePointer
+
+    override init(seed: Data32) {
+        var cc20ptr: OpaquePointer?
+        seed.asMcBuffer { bytesBufferPtr in
+            cc20ptr = withMcInfallible {
+                mc_chacha20_rng_create_with_bytes(bytesBufferPtr)
+            }
+        }
+        ptr = withMcInfallible {
+            cc20ptr
+        }
+        super.init(seed: seed)
+    }
+
+    override var wordPos: Data16 {
```

Comment:
I feel like this is too much "work"/complexity for a computed variable, should be a function, and that also allows for a `throw` modifier (if needed, see above comment).

Reply:
Moved to function - will update with throw if needed when addressing prior comment